### DISCUSSION TITLE
Menu dial rotation direction

### DISCUSSION THREAD

**Discussion by User-462** [srozum/film_camera_tester#49] [Q&A]
@srozum On my unit the menu selection moves to the next item down the list when I turn the knob counter-clockwise. Is this the expected behavior?

**Comment by srozum** [comment]
You mean "natural scroll" doesn't fell natural to you?

**Reply by User-462** [reply]
No. Turn to the right feels more natural when advancing, left to reverse. Like a car radio that advances to higher stations when you turn the knob right. In dialog boxes, Next/Previous = Right/Left, etc.  When I turn the knob on this tester I get it wrong every time if I'm not thinking about it.

**Comment by srozum** [comment]
> Like a car radio that advances to higher stations when you turn the knob right.

Correct. Right turn (clockwise) should select higher menu, left turn (counter-clockwise) should scroll down the menu. And for subselections, like speed of EV, left turn reduces value and right turn increases value.

If you have opposite behavior you probably have different encoder. Tester uses basic type of encoder, like one used in common KY-040 modules.

**Reply by User-462** [reply]
The behavior is as you say, so my encoder and wiring are according to design. 
I agree that turning to the right should increase the EV numbers. 

On the menu however, we are moving the cursor, not the menu. the menu only moves to keep up with the cursor. It would feel more natural to me if I turn the knob right and the cursor moves down to the next item.

**Comment by srozum** [comment]
Check your keypad cable, maybe you switched wires 4 and 5. Or, If you confirm that you have wrong encoder, then try to switch wires 4 and 5.

**Comment by srozum** [comment]
Well, it is as it is.
